status.go

```diff
@@ -41,6 +43,8 @@ func newStatusCollector(cl statusClient) *statusCollector {
 		indexLastModifiedDesc:           prometheus.NewDesc("paperless_status_index_last_modified_timestamp_seconds", "Number of seconds since 01.01.1970 since the last time the index has been modified.", nil, nil),
 		classifierStatusDesc:            prometheus.NewDesc("paperless_status_classifier_status", "Status of the classifier. 1 is OK, 0 is not OK.", nil, nil),
 		classifierLastTrainedDesc:       prometheus.NewDesc("paperless_status_classifier_last_trained_timestamp_seconds", "Number of seconds since 01.01.1970 since the last time the classifier has been trained.", nil, nil),
+		sanityCheckStatusDesc:           prometheus.NewDesc("paperless_status_sanity_check_status", "Status of the sanity check. 1 is OK, 0 is not OK.", nil, nil),
+		sanityCheckLastRunDesc:          prometheus.NewDesc("paperless_status_sanity_check_last_run_timestamp_seconds", "Number of seconds since 01.01.1970 since the last time the sanity check has been run.", nil, nil),
```

Comment:
`January 1, 1970` or `1970-01-01` for a less confusing format.

My reading of `sanity_check_last_run` in Paperless' code is that it's an absolute timestamp. The description currently reads as if it's a relative counter.

Maybe `Most recent sanity check run in seconds since Unix epoch (Jan 1, 1970)`?